.clang-format

```diff
@@ -15,17 +15,21 @@ SpaceBeforeCpp11BracedList: true
 InsertBraces: true
 IncludeBlocks: Regroup
 IncludeCategories:
-  - Regex: '^("|<)llzk/.*("|>)$'  # LLZK headers
-    Priority: 0                   # - Priority 0 forces LLZK headers to be grouped with the "main" header for a file,
-                                  # see https://clang.llvm.org/docs/ClangFormatStyleOptions.html#includecategories
-  - Regex: '^<mlir/.*>$'          # MLIR headers
-    Priority: 1
-  - Regex: '^<llvm/.*>$'          # LLVM headers
-    Priority: 2
-  - Regex: '^<.*(\.h)?>$'         # Standard library headers
+  - Regex: '^("|<)llzk/.*("|>)$'   # LLZK headers
+    Priority: 0                    # - Priority 0 forces LLZK headers to be grouped with the "main" header for a file,
+                                   # see https://clang.llvm.org/docs/ClangFormatStyleOptions.html#includecategories
+  - Regex: '^("|<)llzk-c/.*("|>)$' # LLZK C API headers 
+    Priority: 2                    # Set to priority 2 and onwards in preparation for putting LLZK headers in priority 1
+  - Regex: '^<mlir/.*>$'           # MLIR headers
```

Comment:
Heads up, there's going to be a merge conflict with https://github.com/Veridise/llzk-lib/pull/141/files#diff-1026e0038b722990204a42bed8a6f7c0ec2302aa79e3fad1959d62ba968edfa2 from @shankarapailoor and will need to increment all priorities again when resolving that. Could get ahead of it now by incorporating the `r1cs` rule here.